bitex/interface/bitstamp.py

```diff
@@ -77,136 +77,129 @@ def bid(self, pair, price, size, *args, market=False, **kwargs):
 
     def _place_order(self, pair, price, size, side, market=None, **kwargs):
         """Place an order with the given parameters."""
-        payload = {'amount': size, 'price': price}
-        payload.update(kwargs)
-        if market:
-            return self.request('%s/market/%s/' % (side, pair), authenticate=True, params=payload)
-        return self.request('%s/%s/' % (side, pair), authenticate=True, params=payload)
+        kwargs.update({'amount': size, 'price': price})
+        suffix = 'market/' if market else ''
+        return self.request('%s/%s%s/' % (side, suffix, pair), authenticate=True, params=kwargs)
 
     @format_with(BitstampFormattedResponse)
     def order_status(self, order_id, *args, **kwargs):
         """Return the order status for the given order's ID."""
-        payload = {'id': order_id}
-        payload.update(kwargs)
-        return self.request('api/order_status/', authenticate=True, params=payload)
+        kwargs.update({'id': order_id})
+        return self.request('api/order_status/', authenticate=True, params=kwargs)
 
     @format_with(BitstampFormattedResponse)
     def open_orders(self, *args, pair=None, **kwargs):
         """Return all open orders."""
-        if pair:
-            return self.request('open_orders/%s/' % pair, authenticate=True, params=kwargs)
-        return self.request('open_orders/all/', authenticate=True, params=kwargs)
+        suffix = pair if pair else 'all'
+        return self.request('open_orders/%s/' % suffix, authenticate=True, params=kwargs)
 
     @format_with(BitstampFormattedResponse)
     def cancel_order(self, *order_ids, **kwargs):
         """Cancel existing order(s) with the given id(s)."""
         results = []
-        payload = kwargs
         for oid in order_ids:
-            payload.update({'id': oid})
-            r = self.request('cancel_order/', authenticate=True, params=payload)
+            kwargs.update({'id': oid})
+            r = self.request('cancel_order/', authenticate=True, params=kwargs)
             results.append(r)
         return results if len(results) > 1 else results[0]
 
     @format_with(BitstampFormattedResponse)
-    def wallet(self, *args, **kwargs):
+    def wallet(self, pair=None, *args, **kwargs):
```

Comment:
please keep `args` and `kwargs` in order

Reply:
done